src/hooks/useCioAutocomplete.ts

```diff
@@ -1,12 +1,17 @@
-import { useState } from 'react';
+import { useMemo, useState } from 'react';
 import useCioClient from './useCioClient';
 import useDownShift from './useDownShift';
-import { CioAutocompleteProps, CioClientConfig, UserDefinedSection } from '../types';
+import { CioAutocompleteProps, CioClientConfig, Section, UserDefinedSection } from '../types';
 import usePrevious from './usePrevious';
-import { getItemPosition, getSearchSuggestionFeatures } from '../utils';
+import {
+  getItemPosition,
+  getItemsForActiveSections,
+  getSearchSuggestionFeatures,
+  trackRecommendationView,
+} from '../utils';
 import useConsoleErrors from './useConsoleErrors';
 import useSections from './useSections';
-import useItems from './useItems';
```

Comment:
Removed useItems and added it to utils as a function called getItems instead since there was no need for it to be a hook in the first place